ml-algorithms/src/test/java/org/opensearch/ml/engine/tools/MLModelToolTests.java

```diff
@@ -12,6 +12,8 @@
 import static org.opensearch.ml.engine.tools.MLModelTool.DEFAULT_DESCRIPTION;
 
 import java.util.*;
```

Comment:
help remove this import *?

Reply:
Is there any policy to band any use of *?

Reply:
Done